ui/src/tool-panel/km-post/km-post-infobox.tsx

```diff
@@ -49,6 +56,38 @@ type KmPostInfoboxProps = {
     changeTimes: ChangeTimes;
 };
 
+const CoordinateSystemField: React.FC<{
+    kmPostCrs: CoordinateSystem | undefined;
+    planCrs: CoordinateSystem | undefined;
+}> = ({ kmPostCrs, planCrs }) => {
```

Comment:
Hitusen rikastettu koordinaatistokenttä. Kentän perässä näytetään huomiohuutomerkkineliö jos KM-pylväs on muunnettu toiselta GK-kaistalta ja keltainen varoituskolmio jos sijainti on muunnettu ei-GK-koordinaatistosta. En viitsinyt tehdä näitä osaksi `CoordinateSystemView`:iä, sillä tämä on toistaiseksi haluttua vain täällä